### PR TITLE
adding a section about URL in the test-service-load README

### LINKED ISSUE #5563: Real Service Tests - Facilitate debugging the .fluid files generated from a test run identified in the logs
## Work Item
**Describe the outcome you expect**
If we find a particular event sequence or error in the logs or in the pipeline output for a Real Service Test (e2e or stress test), we would like to be able to quickly jump to that file in SharePoint.

**Approach**
First identify if it's possible to find the file (is it logged by OdspTestDriver?). If yes - how do we make it more streamlined.  If no, how do we log it to make it streamlined.

**Open questions**
* Is it already logged somewhere? How will we correlate to test tenant info/credentials?
* What about r11s tests?  Not relevant in the short term, right?

**Acceptance criteria**
Given some logs from a Real Service Test, we can go find the file in SharePoint and open it with Fetch Tool

### PR DESCRIPTION
**Pull request by chensixx**
Fixes #5563 
When running the stress tests, the URL printed in console can be passed in as-is to fluid debugger and fetch-tool

### CODE REVIEW

File: packages/test/test-service-load/README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ There are several npm scripts in [package.json](./package.json) to make it quick
 `npm run start` - Launches in Orchestrator Mode with default options
 `npm run debug` - Debugs in Orchestrator Mode with `--debug` provided to allow for attaching to child test runners.
 
+### URL
+
+When running the stress tests, there will be a url printed in console, after the line "Connecting to new Container targeting with url:".
```
Comment (tylerbutler) [inline packages/test/test-service-load/README.md:45]:
```suggestion
When running the stress tests, there will be a URL printed in console, after the line "Connecting to new Container targeting with url:".
```

File: packages/test/test-service-load/README.md [hunk 2]
```diff
@@ -40,6 +40,11 @@ There are several npm scripts in [package.json](./package.json) to make it quick
...
+This URL can be passed as-is to Fluid Debugger as well as fetch-tool
```
Comment (tylerbutler) [inline packages/test/test-service-load/README.md:46]:
```suggestion
This URL can be passed as-is to Fluid Debugger as well as fetch-tool.
```

**Review by tylerbutler — APPROVED** [review]
:shipit:

**Review by markfields — APPROVED** [review]
🎂 Thanks!

Side note - Looking at this file overall, I think it needs some updates based on recent refactoring by @anthony-murphy.

**Pull request merged**